src/helpers/logger.ts

```diff
@@ -21,6 +21,10 @@ if (
   logger.add(new winston.transports.MongoDB(mongo))
 }
 
+if (config.get('node.env') === 'test') {
+  logger.add(new winston.transports.Console())
+}
```

Comment:
suggestion: this might be needed somewhere else in the code later. I think it can be extracted to a utils file so we can call it like
```ts
if (isInTestEnv()) {
  ...
}
```

while the utils file contains
```ts
export const isInTestEnv = () => config.get('node.env') === 'test'
```

Reply:
for now it is still to check node_env test is still in this file, will be separated if used elsewhere